sqlite-db/src/impls.rs

```diff
@@ -0,0 +1,44 @@
+use model::CfdEvent;
+
+impl crate::CfdAggregate for model::Cfd {
```

Comment:
❓ I'm surprised that we had to move this here. Why was it?

I don't think we should have and `impls` module in the database layer. This crate provides some APIs which require the implementation of certain traits to be used. It should not need to know any details about its implementers.

Reply:
We have to move this because we can't implement `CfdAggregate` for `model::CFD` in daemon. 

The alternative would be to add the newly created sqlite db crate as dependency to `model`.

Reply:
The database uses the model here not the other way round, so they way this was moved is correct!

Reply:
This is correct. It just feels a little bad to me.